packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -306,16 +307,11 @@ export default defineComponent({
 			};
 		},
 		isCredentialTestable(): boolean {
-			// Sharees can always test since they can't see the data.
-			if (!this.credentialPermissions.isOwner) {
-				return true;
-			}
 			if (this.isOAuthType || !this.requiredPropertiesFilled) {
 				return false;
 			}
 
-			const { ownedBy, sharedWithProjects, ...credentialData } = this.credentialData;
-			const hasUntestableExpressions = Object.values(credentialData).reduce(
+			const hasUntestableExpressions = Object.values(this.credentialData).reduce(
```

Comment:
We now have `homeProject` instead of `ownedBy` - won't this break something?

Reply:
I don't know why those were destructured anyway. Later, in the reduce method, condition checking is done only for the `string` values of the credential object and also checks whether those are expressions.
`(typeof value === 'string' && isExpression(value) && !isTestableExpression(value))`

It shouldn't break anything